docs/content/how-to/altering-tables.md

```diff
@@ -114,6 +114,15 @@ The following SQL adds two columns `c1` and `c2` to table `my_table`.
 
 {{< tabs "add-columns-example" >}}
 
+{{< tab "Flink" >}}
+
+```sql
+CREATE TABLE T (a STRING, b DOUBLE, c FLOAT);
+ALTER TABLE T ADD d INT;
+```
```

Comment:
Please not the words above: The following SQL adds two columns `c1` and `c2` to table `my_table`.
The example should be similar to spark's example.
Flink also support add multiple columns. See (from flink doc):
 ADD { <schema_component> | (<schema_component> [, ...]) }